assignments/assignment02.rb

```diff
@@ -45,7 +75,15 @@ def median(ary)
 
 # implement method `pluck` on array of hashes
 def pluck(ary, key)
-  # your implementation here
+
+  new_array = Array.new
+
+  ary.each do |hash|
+    new_array << hash[key]
+  end
+
+  return new_array
+
```

Comment:
Much more elegant to use map here. You can get rid of creating a new Array, since map does that for you. It just becomes a one liner:
ary.map {|x| x[key]}